### PR TITLE
[MSHARED-860] deprecated Windows 9X CommandShell

### PR DESCRIPTION
**Pull request by elharo**
@michael-o

### CODE REVIEW

File: src/main/java/org/apache/maven/shared/utils/cli/shell/Shell.java [hunk 1]
```diff
@@ -29,12 +29,11 @@
 /**
  * <p>
  * Class that abstracts the Shell functionality,
- * with subclases for shells that behave particularly, like
+ * with subclasses for shells that behave particularly, like</p>
```
Comment (michael-o) [inline src/main/java/org/apache/maven/shared/utils/cli/shell/Shell.java]: Javadoc 8 will probably bug you for this. They expect `<p>` only.

Comment (elharo) [reply src/main/java/org/apache/maven/shared/utils/cli/shell/Shell.java]: fixed

File: src/main/java/org/apache/maven/shared/utils/cli/shell/CmdShell.java [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
 /**
  * <p>
- * Implementation to call the CMD Shell present on Windows NT, 2000 and XP
+ * Implementation to call the CMD Shell present on Windows NT, 2000, X, 7, and 10.
```
Comment (michael-o) [inline src/main/java/org/apache/maven/shared/utils/cli/shell/CmdShell.java]: X => XP
As well as Windows 8

File: src/main/java/org/apache/maven/shared/utils/cli/shell/CommandShell.java [hunk 3]
```diff
@@ -26,8 +26,9 @@
  * </p>
  *
  * @author <a href="mailto:carlos@apache.org">Carlos Sanchez</a>
- * 
+ * @deprecated Windows Me is long dead. Update to Windows 10 and use {@link CmdShell}.
```
Comment (michael-o) [inline src/main/java/org/apache/maven/shared/utils/cli/shell/CommandShell.java]: It was Windows ME. Horrible OS.

File: src/main/java/org/apache/maven/shared/utils/cli/shell/CmdShell.java (hunk 2 above)
Comment (elharo) [reply src/main/java/org/apache/maven/shared/utils/cli/shell/CmdShell.java]: done

File: src/main/java/org/apache/maven/shared/utils/cli/shell/CommandShell.java (hunk 3 above)
Comment (elharo) [reply src/main/java/org/apache/maven/shared/utils/cli/shell/CommandShell.java]: done

**Comment by jira-importer** [comment]
Resolve #305

**Pull request merged**